activesupport/lib/active_support/descendants_tracker.rb

```diff
@@ -27,19 +40,32 @@ def descendants(klass)
           klass.descendants
         end
 
-        def clear(only: nil) # :nodoc:
-          # noop
+        def clear(classes) # :nodoc:
```

Comment:
The only usage left pass `only:` and it's private API, so might as well only support that.